docstrings/gen.go

```diff
@@ -485,6 +485,10 @@ to all instances running in a specific region using the --region/-r flag.`,
 		return KeyStrings{"stop <id>", "Stop a Fly machine",
 			`Stop a Fly machine`,
 		}
+	case "machine.status":
```

Comment:
make test is deleting your changes to this file cause it's generated from this file:
https://github.com/superfly/flyctl/blob/b117369194768c80f0d31c4303518fcf296b36d5/helpgen/flyctlhelp.toml#L1009-L1040